INSTALL.rst

```diff
@@ -224,6 +224,38 @@ To check which JVM will be used by ixmp, run the following in any prompt or term
     $ python -c "import jpype; print(jpype.getDefaultJVMPath())"
 
 
+“No module named 'pyam'”
+------------------------
+
+When running a test in ``message_ix`` an error like this may occur.
```

Comment:
These are all the correct details. For clarity, I'd suggest something like:
```
The package `pyam-iamc <https://pypi.org/project/pyam-iamc/>`_ is one of the "reporting" extra dependencies of :mod:`message_ix`.
These extra dependencies are not installed automatically, but can be installed using::

    # If message_ix is installed using pip
    $ pip install message_ix[reporting]
    # or
    $ pip install pyam-iamc

    # If message_ix is installed using Anaconda
    $ conda install message_ix[reporting]
    # or (see note below)
    $ conda install pyam

Note that this package has the *different* name on conda-forge versus PyPI: `pyam <https://anaconda.org/conda-forge/pyam>`__.

The package listed as `pyam <https://pypi.org/project/pyam/>`__ on PyPI (and not available via Anaconda) is unrelated to :mod:`message_ix`, not compatible with it, and will produce other error messages.
If you installed this package accidentally, remove it using::

    # If installed using pip
    $ pip uninstall pyam
```
The thoughts behind these edits:
- Start with the correct thing to do to resolve the error.
- Don't mention the context of "want to run a test" since there are other possible contexts in which this error could be encountered.
- Then address a common mistake.
- Describe the idea of dependencies without referring to or linking to specific files; these will be inscrutable for most users who run into this error.
- I also put the pip code before the conda code deliberately, because the name (pyam-iamc) aligns. *After* explaining that, the next/separate point is to explain that the name differs on conda-forge vs. on PyPI.
- Remove `conda uninstall pyam` since this would actually remove the desired package, due to the differing names.

Also:
- Per the style guide, text in .rst files is 1 sentence per line.

Reply:
Thank you for your time to review this and your input!